thriftbp/prometheus.go

```diff
@@ -0,0 +1,47 @@
+package thriftbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	serviceLabel             = "thrift_service"
+	methodLabel              = "thrift_method"
+	successLabel             = "thrift_success"
+	exceptionLabel           = "thrift_exception_type"
+	baseplateStatusLabel     = "thrift_baseplate_status"
+	baseplateStatusCodeLabel = "thrift_baseplate_status_code"
+)
+
+var (
+	ThriftLabels = []string{
```

Comment:
do we really need to export these variables? what are their use cases outside of this package? if you only need to access them in the test, you can convert the test to internal test instead (move the test for `PrometheusMetricMiddleware` to a new `_test.go` file and don't add `_test` to the package name of the test file).

Reply:
yes they are only exported for tests. I will move the test file to fix.

Reply:
we might want to actually export the `const`-s with documentation comments so they show up in the documentation?